packages/server/src/constants/index.ts

```diff
@@ -37,3 +37,6 @@ export const paths = (isServer = false) => {
 		REGISTRY_PATH: `${BASE_PATH}/registry`,
 	};
 };
+
+export const GPU_ENABLED = process.env.GPU_ENABLED === 'true';
+export const GPU_RESOURCE_NAME = 'DOCKER_RESOURCE_GPU';
```

Comment:
What is the purpose of this Envs? I don't see any reference

Reply:
hey thanks, nice catch I'll fix this.
> this was there when i was trying to implement GPU enable checkbox at Application level deployment, which is not required now.